google/cloud/bigtable/instance_admin_new_test.cc

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Google Inc.
```

Comment:
s/Inc./LLC/

Reply:
It is temporary. I am not doing things a great way, but in the end it will not matter.

I am going to merge and make the s/Inc./LLC/ in a subsequent PR so I don't have to wait for builds.